LEGO1/mxentity.cpp

```diff
@@ -1,19 +1,15 @@
 #include "mxentity.h"
 
-// OFFSET: LEGO1 0x1001d190 STUB
+// OFFSET: LEGO1 0x1001d190
 MxEntity::MxEntity()
 {
-  // TODO
+  // FIXME: This won't match because of m_atom. How would it access m_internal?
+  this->m_atom = NULL;
+  this->m_mxEntityId = -1;
 }
 
-// OFFSET: LEGO1 0x1000c110 STUB
+// OFFSET: LEGO1 0x1000c110
 MxEntity::~MxEntity()
 {
-  // TODO
-}
-
-// OFFSET: LEGO1 0x10001070 STUB
-void MxEntity::Destroy()
-{
-  // TODO
-}
+  *this->m_atom;
```

Comment:
This destructor is almost certainly empty, the `Unwind`s are the compiler calling the destructors of the member MxAtomId and the base MxCore.

Reply:
Today and tomorrow I'm busy, so hopefully late tomorrow or Thursday I can figure this out, I'm missing something here. 
![image](https://github.com/isledecomp/isle/assets/24401303/47e1adef-b94d-49fa-adc3-544f471d18ff)

Reply:
If you have the MxAtomId as a member, as mentioned above, this will work itself out automatically (the compiler will automatically call its destructor).